src/Sarif.FunctionalTests/DirectProducerTestData/WebSkim/HtmlAttributeAnalysisFixes.sarif

```diff
@@ -0,0 +1,288 @@
+{
+  "$schema": "http://json.schemastore.org/sarif-1.0.0",
+  "version": "1.0.0",
+  "runs": [
+    {
+      "id": "de711026-6d1c-4760-a2ca-c0d94b93312b",
+      "tool": {
+        "name": "ModernCop",
+        "fullName": "mod50 7.0.7000.0",
+        "version": "7.0.7000.0",
+        "semanticVersion": "7.0.7000",
+        "fileVersion": "7.0.20811.0",
+        "language": "en-US",
+        "properties": {
+          "CompanyName": "Microsoft Corporation"
+        }
+      },
+      "results": [
+        {
+          "ruleKey": "WEB1067.AttributeNameIsNotLowercase",
+          "level": "error",
+          "formattedRuleMessage": {
+            "formatId": "Default",
+            "arguments": [
+              "ALT",
+              "area"
+            ]
+          },
+          "locations": [
+            {
+              "analysisTarget": {
+                "uri": "file:///d:/src/SecDevTools/Main/src/ModernCop/Source/WebRules.Test.Unit/ModernCopTestData/WebRules_HtmlRules/Attributes/AttributeAnalysis.html",
+                "region": {
+                  "startLine": 17,
+                  "startColumn": 54,
+                  "endColumn": 57,
+                  "offset": 561,
+                  "length": 3
+                }
+              }
+            }
+          ],
+          "fixes": [
+            {
+              "description": null,
```

Comment:
Shouldn't emit a null (and I'm still arguing for `description` being required), as you mentioned in your email).